src/vtrasterizer/Pixmap.h

```diff
@@ -186,37 +186,37 @@ constexpr void drawEllipseArc(PutPixel putpixel, ImageSize imageSize, crispy::Po
 /// Alpha-channel 2D image.
 struct Pixmap
 {
-    atlas::Buffer _buffer {};
-    ImageSize _size {};
-    ImageSize _downsampledSize {};
-    std::function<int(int, int)> _filler = [](int, int) {
+    atlas::Buffer buffer {};
+    ImageSize size {};
+    ImageSize downsampledSize {};
+    std::function<int(int, int)> filler = [](int, int) {
         return 0xFF;
     };
-    int _lineThickness = 1;
-    int _baseLine = 0; // baseline position relative to cell bottom.
+    int lineThickness = 1;
+    int baseLine = 0; // baseline position relative to cell bottom.
 
-    [[nodiscard]] constexpr ImageSize downsampledSize() const noexcept { return _downsampledSize; }
+    [[nodiscard]] constexpr ImageSize getdownsampledSize() const noexcept { return downsampledSize; }
```

Comment:
Hmm, since it's a struct with all data members being public, we do not need this accessor at all, I think it's even better to simply use `Pixmap.downsampledSize` then, and getting rid of the accessor member methods all together that simply return a data field. What do you think?

Reply:
Done